iac/web/s3.tf

```diff
@@ -64,7 +108,8 @@ resource "aws_s3_bucket_cors_configuration" "asset_bucket" {
     allowed_origins = [
       "http://localhost:4200",
       "http://localhost:3000",
-      "https://${local.buckets.asset.name}"
+      "https://${local.buckets.asset.name}",
```

Comment:
localhost and s3 url are allowed only on `Dev` env

Reply:
Updated